platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/util/BootstrapUtils.java

```diff
@@ -42,19 +130,129 @@
 public final class BootstrapUtils {
 
     /** The logger for this class */
-    private static final Logger logger = LogManager.getLogger(BootstrapUtils.class);
+    private static final Logger LOG = LogManager.getLogger(BootstrapUtils.class);
 
     private BootstrapUtils() {}
 
     /**
-     * Start log4j.
+     * Load the config for paths
+     * @return the paths configuration files
      */
-    public static void startLoggingFramework(final Path log4jPath) {
-        if (log4jPath != null && Files.exists(log4jPath)) {
-            Log4jSetup.startLoggingFramework(log4jPath);
+    public static @NonNull PathsConfig loadPathsConfig() {
+        final PathsConfig pathsConfig = ConfigurationHolder.getConfigData(PathsConfig.class);
```

Comment:
Please pass the config in the method instead of using ConfigurationHolder

Reply:
this is what we spoke about, to build config you need the paths, but to get the paths you need config. ideally, this would be a command line argument

Reply:
Yeah... where you get the config file can't be in the config file itself without causing problems.